atlp-rwanda/e-comm-team-brogrammers-fn
# #184948166 view and edit user profile

**What does this PR do?**
Users should be able to view and edit their profiles
 **Description of Task to be completed?**
as the user logged in should be able to see his/ her profile which include Personal Information name, email, gender, birthdate,
preferred language, preferred currency, and where you live, and be able to change them if they want
 **How should this be manually tested?**
this can be tested manually by visiting brogrammersMall app  then login and you should be able to view and edit profile 
 **Any background context you want to provide?**
n/a

 **What are the relevant pivotal tracker/Trello stories?**
https://www.pivotaltracker.com/story/show/184948166
 **Screenshots (if appropriate)**

 **Questions:**
n/a
